.gitignore

```diff
@@ -19,7 +19,7 @@ m4/lt~obsolete.m4
 autom4te.cache
 
 # downloaded files
-./gmock
+/gmock
```

Comment:
What's the difference here?

Reply:
The ./gmock syntax didn't seem to work for me--I think if you want to match only directories at the top level then you have to put a leading slash but no leading dot.